script.js

```diff
@@ -1,4 +1,109 @@
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  console.log(`Round is ${roundCounter}`);
+  if (roundCounter == `player1Rolls`) {
+    for (var counter = 0; counter < 2; counter += 1) {
+      var roll = dice[Math.floor(Math.random() * dice.length)];
```

Comment:
unique implementation of a dice to store all the sides into an array and get a random element from that array but is there a reason you did it this way rather than just the conventional:

var roll = Math.floor(Math.random() * 6)

nth wrong with it ofc, good experimentation!